scripts/rust.env

```diff
@@ -14,7 +14,7 @@ COVERAGE="-C instrument-coverage"
 FUZZ_SANITIZERS="-Z sanitizer=address,leak"
 OPTIMIZE_OFF="${DEBUG_ASSERTIONS_ON} ${OVERFLOW_CHECK_ON}"
 OPTIMIZE_ON="-C opt-level=3 ${LTO} ${DEBUG_ASSERTIONS_OFF} ${OVERFLOW_CHECK_OFF}"
-OPTIMIZE_FUZZ="-C opt-level=3 ${DEBUG_ASSERTIONS_ON} ${OVERFLOW_CHECK_ON} ${FUZZ_SANITIZERS} ${COVERAGE}"
+OPTIMIZE_FUZZ="-C opt-level=3 ${LTO} ${DEBUG_ASSERTIONS_ON} ${OVERFLOW_CHECK_ON}"
```

Comment:
Just double-checking, is it expected that the `${FUZZ_SANITIZERS}` are not longer part of `OPTIMIZE_FUZZ`?